superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -121,16 +121,16 @@ const getEntityUrl = (entity: ActivityObject) => {
 
 const getEntityLastActionOn = (entity: ActivityObject) => {
   // translation keys for last action on
-  const LAST_VIEWED = `Viewed %s`;
-  const LAST_MODIFIED = `Modified %s`;
+  const LAST_VIEWED = 'Viewed %s';
+  const LAST_MODIFIED = 'Modified %s';
 
   // for Recent viewed items
   if ('time_delta_humanized' in entity) {
-    return t(LAST_VIEWED, entity.time_delta_humanized);
+    return t('Viewed %s', entity.time_delta_humanized);
```

Comment:
I'm not sure this change is necessary - I assume the variable value should be picked up here. Did you check if that's not the case?

Reply:
I verified, but using constants, the string was not extracted. https://github.com/gilbsgilbs/babel-plugin-i18next-extract/issues/109 seems to confirm this, though the FAQ only speaks of variables, not constants.

Reply:
In addition, this whole section seems to rely on `time_delta_humanized` and `changed_on_delta_humanized` being localized, which apparently is not the case:

<img width="780" alt="grafik" src="https://user-images.githubusercontent.com/2187389/143026672-382ffdd7-ae70-404a-9fed-c201a2cef709.png">


For other tables/columns, this seems to be handled differently:
<img width="650" alt="grafik" src="https://user-images.githubusercontent.com/2187389/143026956-27654972-c0f2-4380-9316-45d4374f54e3.png">

Reply:
Oh interesting. Maybe I need to switch my local devenv to non-english to catch these, too. It's becoming apparent we need a proper developer tutorial for i18n, as it's obvious this is not being handled consistently across the app.

Reply:
To avoid duplication and risk of these falling out of sync, maybe we should introduce a similar localized translation mapping as I did in the previous PR: https://github.com/apache/superset/blob/bdc6a1d4e37fd47948752836a7f4f400283b436c/superset-frontend/src/views/CRUD/welcome/EmptyState.tsx#L25-L30

Regarding `time_delta_humanized`, I'm pretty sure these are happening in the backend, let me look at those (I have a faint memory of when those were introduced).